cmd/init.go

```diff
@@ -19,13 +21,22 @@ $ mp init "org:Clever filename:circle.yml"
 would target all Clever repos with a circle.yml file.
 
 See https://help.github.com/articles/searching-code/ for more details about the syntax.`,
```

Comment:
is there any equivalent gitlab URL you'd recommend? e.g. https://docs.gitlab.com/ee/user/search/advanced_global_search.html

Reply:
done